### PR TITLE
Edge 615 - Updated Errors for MFA

### PR DESCRIPTION
**Pull request by psherk-bw**
# **IMPORTANT**
We are slotting breaking changes for February 3rd, and this should go in-tandem with that release. Please reach out to @acovati-bw  or @carrington-bw  with any questions

## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes
Updating the MFA documentation to be up-to-date with current API
- Associated PR: https://github.com/Bandwidth/external-api-docs/pull/26

### CODE REVIEW

File: mfa/errors.md [hunk 1]
```diff
@@ -1,15 +1,239 @@
-# Errors
+{% multimethod %}
+{% endmultimethod %}
 
-Errors are based on the [Voice API Errors](../voice/errors.md) and the [Messaging API Errors](../messaging/errors/about.md).
+# HTTP Errors when using MFA {#top}
 
-## Error Responses
+Bandwidth will respond with HTTP error codes when there are any issues with . Any `4xx` or `5xx` HTTP response code will not be billed. All of the errors except for `401` and `403` have a similar response format. Unifying all of the responses is in the works, so be sure to keep checking the page for updates!
```
Comment (carrington-bw) [inline mfa/errors.md]: The first sentence isn't complete

Comment (psherk-bw) [reply mfa/errors.md]: Thanks for catching that, missed putting MFA

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-edge-615.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-edge-615.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-edge-615.s3-website-us-east-1.amazonaws.com/index.html

**Review by jmulford-bw — APPROVED** [review]

**Pull request merged**